packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -551,55 +551,80 @@ describeIf(process.platform !== 'win32' && !isMacOrWindowsCI)('MongoDB', () => {
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
 
+  test('introspection --force', async () => {
+    ctx.fixture('schema-only-mongodb')
+    const introspect = new DbPull()
+    await introspect.parse(['--force'])
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchSnapshot()
```

Comment:
Should we maybe pull in these snapshots as InlineSnapshots as well to make it more obvious what the full result it?